README.md

```diff
@@ -1 +1,173 @@
-# sql-commands-intro
\ No newline at end of file
+# SQL Commands Introduction
+This workshop will teach you how to use basic SQL - Structured Query Language. SQL is a standard for relational database systems, while MySQL and PostgreSQL are implementations of the standard.
+
+## Syntaxes
```

Comment:
This section seems a bit redundant. You have the same information in  "Setting up the workshop database"